src/metabase/task/sync_databases.clj

```diff
@@ -340,7 +340,7 @@
                     (log/warnf e "Error updating database %d for randomized schedules" (u/the-id db))
                     counter))))
              (t2/reducible-query
-              {:select [:id :details]
+              {:select [:*]
```

Comment:
How is this related to the secrets change?

Reply:
adding the after-select for secrets made this test blow up because :engine wasn't selected. If something is going to manually call the after-select we need the full table.